tasks/81_buying-car/buying-car.js

```diff
@@ -0,0 +1,14 @@
+export const nbMonths = (priceOld, priceNew, saveMonth, percLoss ) => {
+  let savings = priceOld;
+  let time = 0;
+
+  while (savings < priceNew) {
+    time++;
+    percLoss = time % 2 === 0 ? 0.5 + percLoss : percLoss;
```

Comment:
generally it is not very good practice to reassign function parameters

Reply:
[Agree](https://eslint.org/docs/rules/no-param-reassign)